packages/serverless/src/domain/use-case/rollup.ts

```diff
@@ -149,16 +151,21 @@ export async function getRollUpHistory(
     .toArray();
 
   const buildRollUpHistory = (
-    history: TRollUpHistory,
-    transaction: TTransaction,
+    history: TRollUpHistoryRecord,
+    transaction: TTransactionRecord,
     status: ETransactionStatus,
-    error?: string
-  ) => ({
+    error: string
+  ): TRollUpTransactionHistory => ({
     databaseName: history.databaseName,
     currentMerkleTreeRoot: history.currentMerkleTreeRoot,
     previousMerkleTreeRoot: history.previousMerkleTreeRoot,
+    transactionObjectId: history.transactionObjectId,
+    proofObjectId: history.proofObjectId,
+    tx: transaction.tx,
```

Comment:
Only use one type name for all, why rename it to `tx`?

Reply:
renamed to txBody